changelogs/fragments/1176-vmware_dv_portgroup

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - vmware_dv_porgroup - added VMware DV portgroup object_type for setting permissions (https://github.com/ansible-collections/community.vmware/pull/1176)
```

Comment:
```suggestion
  - vmware_object_role_permission - added VMware DV portgroup object_type for setting permissions (https://github.com/ansible-collections/community.vmware/pull/1176)
```

The changelog fragment should make it clear that there is a change to the `vmware_object_role_permission` module ;-)